evap/evaluation/templates/contact_modal.html

```diff
@@ -48,6 +48,8 @@ <h5 class="modal-title" id="{{ modal_id }}Label">{{ title }}</h5>
     function {{ modal_id }}Show() {
         $('#{{ modal_id }}ActionButton').unbind().click(function(){ {{ modal_id }}Action(); });
         $('#{{ modal_id }}').modal();
+        // remove wrongly added padding by Bootstrap (https://github.com/twbs/bootstrap/blob/v4-dev/js/src/modal.js#L429)
+        $('body, .navbar, .footer').css('padding-right', '');
```

Comment:
if you think that is a bug, please consider reporting it. there have been previous (closed) reports of this (e.g. [here](https://github.com/twbs/bootstrap/issues/24934)), but still, please consider it.

Reply:
I think it is no bug on their side, the `padding` just does not play well with a `overflow: scroll !important` on the body, so I think it is our fault here